package.json

```diff
@@ -48,11 +48,12 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,json}": [
+    "*.{js,jsx}": [
       "prettier --write",
+      "eslint --fix",
```

Comment:
Yes, that was the intent. I was hoping this would inspire a "clean as you go" approach. I was also hoping to have things in a pretty clean starting place. Hopefully it's not too annoying. Follow-up idea coming...

Reply:
Cool. Just checking :)